mlrun/runtimes/nuclio/serving.py

```diff
@@ -207,7 +201,6 @@ def __init__(
         self.graph_initializer = graph_initializer
         self.error_stream = error_stream
         self.track_models = track_models
-        self.tracking_policy = tracking_policy
```

Comment:
also remove the same line from the `GraphServer` in `mlrun/serving/server.py` (L132)